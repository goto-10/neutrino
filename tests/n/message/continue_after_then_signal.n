import $core;
do {
  $core:delay(fn => $core:print_ln!("a")).then(fn $a => leave.blam! $a);
  $core:delay(fn => $core:print_ln!("b"));
}
