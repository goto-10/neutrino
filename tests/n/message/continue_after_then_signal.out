Error: --- backtrace ---
- leave.blam!("a")
- lambda~(canonicalized)("a")
"a"
"b"
